ActivityTalkbackInvestigation/app/src/main/java/com/novoda/activitytalkbackinvestigation/ProHaxDrawerActivity.java

```diff
@@ -0,0 +1,97 @@
+package com.novoda.activitytalkbackinvestigation;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.annotation.LayoutRes;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+
+public abstract class ProHaxDrawerActivity extends AppCompatActivity {
+
+    private DrawerLayout drawerLayout;
+
+    @Override
+    public void setContentView(@LayoutRes int layoutResID) {
+        super.setContentView(R.layout.drawer_layout);
+        ViewGroup contentView = (ViewGroup) findViewById(R.id.content_frame);
+        getLayoutInflater().inflate(layoutResID, contentView);
+        setupDrawer();
+        drawerLayout.openDrawer(GravityCompat.START);
```

Comment:
this would need to be guarded somehow so it doesn't open the drawer if the activity was opened by the app launcher.